code/ai/aicode.cpp

```diff
@@ -11457,7 +11457,7 @@ void get_absolute_wing_pos(vec3d *result_pos, object *leader_objp, int wingnum,
 
 	if (wing_index == 0)
 		wing_delta = vmd_zero_vector;
-	else if ( formation_index != -1 && wing_index < MAX_SHIPS_PER_WING) 
+	else if ((formation_index >= 0) && (formation_index < (int)Wing_formations.size()) && (wing_index < MAX_SHIPS_PER_WING)) 
```

Comment:
else if (!Wing_formations.empty() && (formation_index >= 0) && (wing_index < MAX_SHIPS_PER_WING))

Reply:
The github suggestions feature isn't working, so use that line instead.

Turns out it's okay for `Wing_formations` to be empty, but it's better to explicitly check for that.

Reply:
If we switch to that, would there be a reason to add an assert to check `formation_index < Wing_formations.size()` ?

Reply:
Yeah, an Assert is reasonable for that.  It shouldn't ever happen, but it's good to safeguard against it.